src/views/Layout.jsx

```diff
@@ -21,15 +21,21 @@ export function Layout() {
 					<Outlet />
 				</main>
 				<nav className="Nav">
-					<NavLink to="/" className="Nav-link">
-						Home
-					</NavLink>
-					<NavLink to="/list" className="Nav-link">
-						List
-					</NavLink>
-					<NavLink to="/add-item" className="Nav-link">
-						Add Item
-					</NavLink>
+					{!listToken && (
```

Comment:
Just purely out of curiosity, is there any reason why you chose to do `&&` here and on line 29, instead of making this one ternary?

Reply:
I hadn't really thought of using a ternary, tbh! I started with JUST the `!listToken` section (having only the <Home/> NavLink show if `!listToken`), and after that was built I added the opposite.